roles/confluent.common/tasks/precheck.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Confirm Hash Merging Enabled
+  assert:
+    that: lookup('config', 'DEFAULT_HASH_BEHAVIOUR') == 'merge'
+    fail_msg: "Hash Merging must be enabled in ansible.cfg: Confirm current directory is cp-ansible and permissions are set to 755 not 777"
+
+- name: Confirm Rhel Version Supported
```

Comment:
Are all the following validations to consider as "hard" requirements? Would it make sense to introduce a variable to supress the more "softer" requirements?

I also think there must be some bugs in the asserts. Please review.

Reply:
These are the "supported os's" we have documented here: https://docs.confluent.io/platform/current/installation/system-requirements.html#sys-req-os

Are you suggesting we give users a back door for skipping these asserts and installing on rhel6 or something?

Reply:
Yes, if some of the requirements are "soft" (might work, but not supported) I think it makes sense.

Reply:
added a flag, lmk what you think